cmd/sos_download.go

```diff
@@ -159,6 +148,32 @@ var downloadCmd = &cobra.Command{
 	},
 }
 
+// destinationExists verifies that the given destination does not exist
+func destinationExists(localFilePath string, objectName string) (bool, error) {
+	st, err := os.Stat(localFilePath)
+	if err == nil {
+		// If the destination exists and is a directory.
+		if st.IsDir() {
+			localFilePath = path.Join(localFilePath, objectName)
+			_, err = os.Stat(localFilePath)
+			if err == nil {
+				return true, nil
+			}
+		} else {
+			return true, nil
+		}
+	}
+
+	// Proceed if file does not exist. return for all other errors.
+	if err != nil {
+		if !os.IsNotExist(err) {
+			return false, err
+		}
+	}
+	return false, nil
+}
+
 func init() {
 	sosCmd.AddCommand(downloadCmd)
+	downloadCmd.Flags().Bool("allow-overwrite", false, "Allow overwriting the destination")
```

Comment:
I'd prefer we go with the usual *parlance* in CLI tools:

```suggestion
	downloadCmd.Flags().BoolP("force", "f", false, "Overwrite the destination file if it already exists")
```

Reply:
Will do. I'll refactor the variable names to `force` as well.